File: specialTopics/introductionVB.Net/heritage/Student.java


public class Student extends Person {

  private String registration;

  Student(String name, String registration, String documents) {
    this.setName(name);
    this.setDocument(document);
    this.registration = registration;
  }

  private void setDocument(String document) {}

  public String getregistration() {
    return registration;
  }

  public void setRegistration(String registration) {
    this.registration = registration;
  }
}

File: specialTopics/introductionVB.Net/heritage/Student.vb
Public Class Student Inherits Person

    Private Registration As String
    
    Sub New(ByVal name As String, ByVal Registration As String, ByVal document As String)
        Me.pname = name
        Me.pdocument = document
        Me.pregistration = Registration
    End Sub

    Property pregistration() As String
        Get
            Return Registration
        End Get
        Set(ByVal value As String)
            Registration = value
        End Set
    End Property
End Class
